week2/runTests.py

```diff
@@ -0,0 +1,7 @@
+import unittest
+loader = unittest.TestLoader()
+start_dir = 'tests'
+suite = loader.discover(start_dir)
+
+runner = unittest.TextTestRunner()
+runner.run(suite)
```

Comment:
I would write this following PEP8 standards:

```
import unittest

START_DIR = 'tests'

loader = unittest.TestLoader()
suite = loader.discover(START_DIR)
runner = unittest.TextTestRunner()

runner.run(suite)
```

In Python the module/file should be named `run_tests.py`.